# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/wasm/WasmTable.cpp
# Commit: 349728105c40
# Full Hash: 349728105c40993896170f4c2f43e152a3466706
# Author: Ryan Hunt <rhunt@eqrion.net>
# Date: 2022-01-08 09:40:24
# Regressor Bug: 1642412
# File Overlap Count: 19
# Description:
#   Bug 1748602 - Backout bug 1642412. r=lth
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D135346
# ==============================================================================

diff -r 32873022d2ed -r 349728105c40 js/src/wasm/WasmTable.cpp
--- a/js/src/wasm/WasmTable.cpp	Fri Jan 07 20:35:15 2022 +0000
+++ b/js/src/wasm/WasmTable.cpp	Fri Jan 07 20:43:16 2022 +0000
@@ -140,9 +140,9 @@
   }
 }
 
-uint8_t* Table::tlsElements() const {
+uint8_t* Table::functionBase() const {
   if (repr() == TableRepr::Ref) {
-    return (uint8_t*)objects_.begin();
+    return nullptr;
   }
   return (uint8_t*)codePtrs_.get();
 }